test/test_distributions.py

```diff
@@ -226,11 +232,23 @@ def _check_enumerate_support(self, dist, examples):
             actual = dist(param).enumerate_support()
             self.assertEqual(actual, expected)
 
+    def test_support_return_type(self):
```

Comment:
nit: Maybe call this `test_enumerate_support_type` since `.support` is a property with different meaning?

Reply:
Done.